src/sider/arbeidsuforhet/ArbeidsuforhetOppfylt.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from "react";
+import { useArbeidsuforhetVurderingQuery } from "@/data/arbeidsuforhet/arbeidsuforhetQueryHooks";
+import { Alert } from "@navikt/ds-react";
+import { VurderingType } from "@/data/arbeidsuforhet/arbeidsuforhetTypes";
+
+const texts = {
+  success:
+    "Begrunnelsen din på at bruker oppfyller 8-4 er lagret i historikken",
```

Comment:
Vet ikke om det ble diskutert noe mer, men Peter snakket jo litt om at han ville at det ikke skulle stå paragrafer rundt omkring. For min del er det greit, men da kanskje `§ 8-4` eller `§ 8-4 arbeidsuførhet`, evt `...oppfyller kravet om arbeidsuførhet`. Og kanskje punktum på slutten? 😊